src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{650277B5-9423-4ACE-BB54-2659995B21C7}</ProjectGuid>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx' OR '$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx' Or '$(TargetGroup)' == 'net46' Or '$(TargetGroup)' == 'netcoreapp' Or '$(TargetGroup)' == 'uap'">true</IsPartialFacadeAssembly>
```

Comment:
I don't think this is a proper usage of IsPartialFacadeAssembly I think for targetgroup != netfx and != net46 you would need to add the typeforwards to System.Runtime manually. That is why the build is failing from AFAIK but not sure,  @weshaggard knows more about the partial facade generation.

Reply:
If we do go forward with this you don't want to turn the contract into a partial facade unless you are going to move the API's to System.Runtime which I don't think we are planning to do.